src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Dependencies/PackageAnalyzersDependencyNode.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using Microsoft.VisualStudio.Imaging;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS.Tree.Dependencies
+{
+    public class PackageAnalyzersDependencyNode : DependencyNode
```

Comment:
>PackageAnalyzersDependencyNode [](start = 17, length = 30)

Do we want an Analyzers node? Most packages that bring analyzers are going to have just analyzers. In that case this would be redundant. For packages that have references and analyzers, the icon differentiates them anyway. Maybe we can get rid of this and keep the depth of the tree smaller?